src/datafactory/azext_datafactory/manual/version.py

```diff
@@ -8,4 +8,4 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.10.0"
+VERSION = "0.11.0"
```

Comment:
We have new version guideline for cli extensions. Maybe you need to upgrade to `1.0.0` now. cc @jsntcy

Reply:
update